zuul.d/jobs.yaml

```diff
@@ -127,6 +127,16 @@
       - name: github.com/ansible/ansible-zuul-jobs
     nodeset: vsrx3-18.4R1-python36
 
+- job:
+    name: ansible-network-vqfx-appliance
+    parent: ansible-network-junos-appliance
```

Comment:
what we should do here is rename ansible-network-junos-appliance, to the actually version of junos. Something like ansible-network-junos-vsrx-appliance, then we create ansible-network-junos-vqfx-appliance.  This means, two top level jobs that we can then parent ansible-test jobs too